components/skeleton/skeleton-mixin.js

```diff
@@ -72,6 +72,12 @@ export const skeletonStyles = css`
 		content: '\\A \\A \\A \\A';
 		white-space: pre;
 	}
+	:host([skeleton][dir="rtl"]) .d2l-skeletize-paragraph-2,
+	:host([skeleton][dir="rtl"]) .d2l-skeletize-paragraph-3,
+	:host([skeleton][dir="rtl"]) .d2l-skeletize-paragraph-5 {
```

Comment:
I ran this through a CSS autoprefixer with our current support settings and the `-webkit-*` prefix isn't needed.